axon-server-connector/src/main/java/org/axonframework/axonserver/connector/ErrorCode.java

```diff
@@ -27,87 +27,167 @@
 import org.axonframework.common.AxonException;
 import org.axonframework.eventsourcing.eventstore.EventStoreException;
 import org.axonframework.messaging.EventPublicationFailedException;
+import org.axonframework.messaging.HandlerExecutionException;
 import org.axonframework.modelling.command.ConcurrencyException;
 import org.axonframework.queryhandling.NoHandlerForQueryException;
 import org.axonframework.queryhandling.QueryExecutionException;
 
-import java.util.function.BiFunction;
+import java.util.function.Supplier;
 
 import static java.util.Arrays.stream;
 
 
 /**
  * Converts an Axon Server Error to the relevant Axon framework exception.
+ *
+ * @author Marc Gathier
+ * @since 4.0
  */
 public enum ErrorCode {
+
     // Generic errors processing client request
-    AUTHENTICATION_TOKEN_MISSING("AXONIQ-1000", AxonServerException::new),
-    AUTHENTICATION_INVALID_TOKEN("AXONIQ-1001", AxonServerException::new),
+    AUTHENTICATION_TOKEN_MISSING("AXONIQ-1000", (code, error, details) -> new AxonServerException(code, error)),
+    AUTHENTICATION_INVALID_TOKEN("AXONIQ-1001", (code, error, details) -> new AxonServerException(code, error)),
 
     //Event publishing errors
-    INVALID_EVENT_SEQUENCE("AXONIQ-2000", (code, error) -> new ConcurrencyException(error.getMessage(), new AxonServerException(code, error))),
-    NO_EVENT_STORE_MASTER_AVAILABLE("AXONIQ-2100", (code, error) -> new EventPublicationFailedException(error.getMessage(), new AxonServerException(code, error))),
-    EVENT_PAYLOAD_TOO_LARGE("AXONIQ-2001", (code, error) -> new EventPublicationFailedException(error.getMessage(), new AxonServerException(code, error))),
+    INVALID_EVENT_SEQUENCE(
+            "AXONIQ-2000",
+            (code, error, details) -> new ConcurrencyException(error.getMessage(), new AxonServerException(code, error))
```

Comment:
Did formatting go off?

Reply:
No, I intentionally new lined some things so that the fitted the line length fully.
Regardless, thanks for being on to it to ensure we follow our style guide, much appreciated!